content/blog/using-jina-embeddings-haystack/index.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: blog-post
+title: 'Using Jina Embeddings v2 with Haystack 2.0 pipelines to summarize legal documents'
+description: Learn how to use the Jina v2 Embedding models in a RAG pipeline with our new Haystack integration.
+featured_image: thumbnail.png
+images: ["blog/using-jina-embeddings-haystack/thumbnail.png"]
+alt_image: The logos for Haystack and Jina AI sit against a blue background with cartoonish art of a man and a woman holding a magnifying glass up to a large clipboard, next to a gavel.
+date: 2024-01-03
+last_updated: 2024-01-03
+authors:
+  - Tilde Thurium
+tags: ["Community", "Embeddings", "Open Source", "Integration"]
+---
+
+[Jina.ai](https://jina.ai/) recently upgraded and expanded the capabilities of their previous embedding model in a v2 release. 
+
+With the [Jina Haystack extension](https://haystack.deepset.ai/integrations/jina), you can now take advantage of these new text embedders in your Haystack pipelines! In this post, we'll show what's cool about Jina Embeddings v2 and how to use them.
+
+> You can follow along in the accompanying [Colab notebook of a RAG pipeline that uses the Jina Haystack extension](https://colab.research.google.com/drive/1l8GbQhqxnWXkdktgJfs9Rz4EAtNbHK_L#scrollTo=_coq_qCuItbN).
```

Comment:
You can add this notebook to the haystack-cookbook repo now

Reply:
opened a pr for that! https://github.com/deepset-ai/haystack-cookbook/pull/9